openwisp_controller/config/static/config/js/vpn.js

```diff
@@ -20,7 +20,7 @@ django.jQuery(function ($) {
             spinner.fadeOut(250, function () {
                 var message = gettext(
                     'Please be patient, we are creating all the necessary ' +
-                        'cyrptographic keys which may take a minute or two ...'
+                        'cyrptographic keys which may take some minutes'
```

Comment:
some time, not some minutes.
It can take a few seconds. Shouldn't take more than that.